src/plugins/api-gateway/cli/inspect-endpoint.js

```diff
@@ -1,103 +1,113 @@
 'use strict';
 
+// Nice ES6 syntax
+// const { Promise, _, icli } = lager.import;
 const lager = require('@lager/lager/lib/lager');
-const Promise = lager.getPromise();
-const _ = lager.getLodash();
-const cliTools = require('@lager/lager/lib/cli-tools');
+const Promise = lager.import.Promise;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["Promise" is defined but never used](https://www.codacy.com/app/lager/lager/file/2779185925/issues/source?bid=3360491&fileBranchId=3365065#l6)